### DISCUSSION TITLE
Check bindings of switch

### DISCUSSION THREAD

**Discussion by Vertikar** [Koenkk/zigbee2mqtt#9705] [Devices]
Is there anyway to check what devices are bound directly to an Ikea E1524 Tradfri remote?
I have two Xiaomi ZNCZ02LM power plugs that somehow seem to have been bound to it, and are being toggled by the switch.
I've shutdown appdaemon and home assistant, and they're still being toggled, so it seems like a binding.

**Comment by palitu** [comment]
Me too!

I cannot figure out how it is triggering ALL of my Xiaomi plugs!  i have other tradfri remotes which dont seem to do this.

I cannot see any bind groups.  I seem to recall something about adding them to a default bind group?  or something like that, but i  am not to sure on the details.

**Comment by palitu** [comment]
ok, i have it figured.  there is some conflicting info, but!  this is what worked for me, with the exact same issue that you have:

1. force remove the E1743 remotes using the front end
2. re-pair them
3. unbind them from the default group:
3.1 Publish an MQTT message to `zigbee2mqtt/bridge/request/device/unbind` with a payload of `{"from":"<device_id>","to": "default_bind_group"}`.  Make sure to push a button on the deivce to wake it up before publishing.
4. Create a new group in the front end (call it something like switch-dump)
5. for the device you have just re-paired, bind it to that group with the `LevelCtl` and `OnOff` clusters ticked.

from there you should be good.

**Reply by pjcarly** [reply]
Looks interesting, what is the device ID in the message? Where can I find it? is it the IEEE id or something else?

**Comment by palitu** [comment]
Just to add further, i just went through and removed ALL of my Xiaomi plugs from the default bind group using the above.  It seems to remove the issue.

**Reply by Vertikar** [reply]
I'll give this a go, as trying your first lot of steps only fixed it temporarily.

**Reply by Vertikar** [reply]
Oh did you add them to a new bind group?

**Reply by palitu** [reply]
> Oh did you add them to a new bind group?

No i just removed them.  My hypothesis is that certain devices send commands, and will react to commands when they are in certain bind groups.  This causes them to have seemingly un-expected behaviour.  

I have found some rocker type switches from IKEA, as well as the MINI-L from Sonoff.  this results in the Xiaomi switches toggling.

Once i removed the switches from the group, the new Sonoff stopped turning off my NAS....  so, this is another assumed solution, but seems much more likely to be permanent for the affected switches.

Note, that i cannot see them joined to the bind group on the Z2M GUI.
